README.md

```diff
@@ -22,3 +22,6 @@ Things you may want to cover:
 * Deployment instructions
 
 * ...
+
+* Contribution Guide
+  - Branch name must follow syntax [firstNameInitial/clikupTicketNumber-short-description]
```

Comment:
i added the brackets for emphasis, so the branch name should not have the brackets

Reply:
okay noted